src/store/store.ts

```diff
@@ -8,8 +8,8 @@ import { componentReducer } from "slices/components";
 import { donation } from "slices/donation";
 import gift from "slices/gift";
 import widget from "slices/widget";
+import amplifyConfig from "constants/aws";
 import { appRoutes } from "constants/routes";
-import config from "../aws-exports";
```

Comment:
*aws-exports.js* is currently unused:
- the file can be removed
- reference to file in *.gitignore*
- reference to file in *.prettierignore*

Reply:
Also there's a [failing build](https://github.com/AngelProtocolFinance/angelprotocol-web-app/actions/runs/7137940363/job/19438687816?pr=2583)